modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+        "score": 0.20427070096000927
+    },
+    {
+        "original_title": "Star Wars: Episode III - Revenge of the Sith",
+        "score": 0.18790171387166904
+    },
+    {
+        "original_title": "Star Wars: Episode II - Attack of the Clones",
+        "score": 0.18790171387166904
+    },
+    {
+        "original_title": "Star!",
+        "score": 0.11436285490763314
+    },
+    {
+        "original_title": "The Star",
+        "score": 0.11436285295039623
+    },
+    {
+        "original_title": "War",
+        "score": 0.10125028723815423
+    },
+    {
+        "original_title": "The War",
+        "score": 0.10125028550532937
+    },
+    {
+        "original_title": "The War",
+        "score": 0.10125028550532937
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.0885850854356994
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.0885850854356994
+    },
+    {
+        "original_title": "A Star for Christmas",
+        "score": 0.0885850854356994
+    }
+    ]
+----
+
+
+*Star Wars* was in first position and now after "improving" our search, it appears just as the 5th result. As expected, adding new fields generated more noise in the score. 
+
+Here is how the overview of *Star Wars* looks like:
+
+[quote,]
+____
+Princess Leia is captured and held hostage by the evil Imperial forces in their effort to take over the galactic Empire. Venturesome Luke Skywalker and dashing captain Han Solo team together with the loveable robot duo R2-D2 and C-3PO to rescue the beautiful princess and restore peace and justice in the Empire
+____
+
+
+And here is the overview of *Robot Chicken: Star Wars*
+
+
+[quote,]
+____
+Fans of Adult Swim's 'Robot Chicken' and the Star Wars movie franchise won't want to miss this collection of 30 sketches. This hilarious compilation features an array of skits -- such as 'Darth Vader's Collect Call' and 'Inside the AT-AT' -- as well as the incredible voice talents of Hulk Hogan, Malcolm McDowell, Conan O'Brien and even the original Luke Skywalker himself, Mark Hamill.
+____
+
+In the Robot Chicken case, *Star Wars* appear both in the title and in the description, that is the reason why it is currently more relevant than the original movie. 
+
+
+=== Manipulating Scores with User Defined Functions
+
+So far, if the user searches for *Star War* (with a missing *S*), he would still get fairly relevant results. However, the top 5 ones are not exactly the most popular Star Wars movies ever created.
+
+We could improve that by manipulating the score. Let's use the attributes *popularity*, *release_year* and *rating* to boost or penalize movies and see how it will affect the final order.
+
+Couchbase 6.5 allows you to create link:https://docs.couchbase.com/server/6.5/n1ql/n1ql-language-reference/userfun.html[User Defined Functions], we will use this feature to create a super naive function to manipulate our score:
+
+[source, Javascript]
```

Comment:
`[source,JSON]`

Reply:
This is actually a javascript function, so I suppose we should keep Javascript